js/components/main.js

```diff
@@ -620,6 +623,16 @@ class Main extends ImmutableComponent {
     windowActions.setUrlBarActive(false)
   }
 
+  onFindHide () {
+    const activeFrame = FrameStateUtil.getActiveFrame(this.props.windowState)
+    windowActions.setFindbarShown(activeFrame, false)
+    webviewActions.stopFindInPage()
```

Comment:
the problem with this is that calling `stopFindInPage` resets the webcontents find state so you can't get back to the same search results